### PR TITLE
testing support for custom names

### PR DESCRIPTION
**Pull request by minidevz**
This might not work since i need to test it in the actual github.io link, but ill fix it if it doesn't :p

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
-
       <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
       <link rel="stylesheet" href="./public/styling/stylesheet.css">
       <link rel="shortcut icon" type="image/png" href="./images/Ballon.png">
+      <script> 
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+      var name = urlParams.get('name')
+      document.getElementById('output').innerHTML = name;  
+      </script>
```
Comment (harshhhdev) [inline index.html:13]: Here, you need to bring this script to the bottom so it does not show a null reference.

**Review by harshhhdev — CHANGES_REQUESTED** [review]
Bring the output to the bottom, so it does not show a null reference.

File: index.html [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
    </head>
 
    <body>
-      <h1 class="title">Happy Birthday Mom! <!--insert the name of person you want to wish here --></h1>
+      <h1 class="title">Happy Birthday</h1> <h1 id="output"></h1>
```
Comment (harshhhdev) [inline index.html:17]: Instead of this, in the name you can do `Happy Birthday ${name}`

**Review by harshhhdev — COMMENTED** [review]
Add a name to the JS code instead of having 2 seperate elements.

File: index.html (hunk 2 above)
Comment (minidevz) [reply index.html:17]: That doesn't work in HTML though.

**Comment by minidevz** [comment]
Made all changes other than the ${name} one (since thats not possible in HTML), should be good to go.

File: index.html [hunk 3]
```diff
@@ -26,4 +24,16 @@ <h1 class="title">Happy Birthday Mom! <!--insert the name of person you want to
          happyBirthdaySong.play()
       }
    </script>
+   <script> 
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+      var name = urlParams.get('name')
+      console.log(name);
+      document.getElementById('output').innerHTML = name + "!";
```
Comment (harshhhdev) [inline index.html:32]: Only one h1 element instead of 2. For output you can do ‘Happy Birthday ${name}’

**Review by harshhhdev — COMMENTED** [review]
> Made all changes other than the ${name} one (since thats not possible in HTML), should be good to go.

Sorry. I should’ve made it more clear what I wanted. I wanted the string interpolation in JS, not the HTML (since it’s not possible to do). No need for 2 h1 tags.

Other than that it’s good 👍🏽Just have only one h1 element and do the string interp in the JS and I’ll merge when I get home from school

File: index.html (hunk 3 above)
Comment (minidevz) [reply index.html:32]: yeah i get it thanks.

**Comment by minidevz** [comment]
I think ive made all the changes you needed, also vertically centered the happy birthday text because it looks better that way, think we should be good to go

**Review by harshhhdev — APPROVED** [review]

**Comment by harshhhdev** [comment]
Yep, thanks for this.

**Pull request merged**